go.mod

```diff
@@ -126,8 +126,8 @@ require (
 
 go 1.20
 
-// Ensure we only pull in the latest version of containerd
-replace github.com/containerd/containerd => github.com/containerd/containerd v1.7.6
+// Ensure we only pull in the latest version of containerd 1.6.x
+replace github.com/containerd/containerd => github.com/containerd/containerd v1.6.24
```

Comment:
Can we remove this in favor of `github.com/containerd/containerd v1.6.24` in the main require block? Or what does that end up downgrading?

Reply:
I wonder if this fixes the issue then - I don't think go tool respects the replace directives when importing as a library.

Reply:
Ohhh... you are right 🤔

Reply:
It looks like [kaniko](https://github.com/GoogleContainerTools/kaniko/blob/main/go.mod#L42) is probably one of the main packages forcing our hand here to a later containerd version. Oh well I suppose.